templates/graphql-js-starter/graphback.json

```diff
@@ -17,5 +17,12 @@
     "subDelete": false,
     "disableGen": false
   },
+  "defaultPaths": {
```

Comment:
Minor question/suggestion. Is `defaultPaths` the name that will be present in the `graphback.json` file that users will modify? As an end user, the name `defaultPaths` is kinda weird because they're not default paths right? They're paths that I can change. Maybe something like `paths` or `files` would make a little more sense.